### PR TITLE
chore: missing button examples added

### LINKED ISSUE #296: Improve Docs: Add Search
### Description
Follow design seen here:
https://sap.github.io/fundamental-ngx/docs/home

image

CORUI-6169

### PR DESCRIPTION
**Pull request by leannerivera**
### Description
Missing medium button examples added:
screen shot 2019-02-13 at 4 39 19 pm
screen shot 2019-02-13 at 4 39 10 pm

### CODE REVIEW

File: src/_playground/App.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@ $fd-fonts-path: '../../node_modules/fiori-fundamentals/dist/fonts/';
 @import '../../node_modules/fiori-fundamentals/scss/all.scss';
 
 // Fundamental React Docs Styles
+
```
Comment (bcullman) [inline src/_playground/App.scss]: @leannerivera - first off congrats on your first PR!

There are some unrelated changes here that should be removed from this PR so that it remains focused.

In App.scss there are some white space diffs that should be removed.

File: src/Button/Button.Component.js [hunk 2]
```diff
@@ -33,6 +33,10 @@ const clickBtnHandler = btn => {
     Approve
```
Comment (bcullman) [inline src/Button/Button.Component.js]: these examples in `Button.Component.js` may be useful, but they are unrelated to search functionality. Remove (and perhaps add back with another chore PR)

File: src/Button/Button.Component.js [hunk 3]
```diff
@@ -86,6 +92,14 @@ const clickBtnHandler = btn => {
     Disabled State
 </Button>
 
+<Button type="medium">Normal State</Button>
```
Comment (jbadan) [inline src/Button/Button.Component.js]: @leannerivera I think we forgot to remove the buttons we were playing with 😂

File: src/Button/Button.Component.js (hunk 2 above)
Comment (leannerivera) [reply src/Button/Button.Component.js]: Oops! Sorry I just submitted search in my practice branch and forgot about them. Will do! :)

File: src/Button/Button.Component.js (hunk 3 above)
Comment (leannerivera) [reply src/Button/Button.Component.js]: Yeah, gonna do that now. xD

**Comment by bcullman** [comment]
looks tight. 🚢

**Comment by bcullman** [comment]
@greg-a-smith agreed on `type`. worse, `type` is part of the normal HTML5 button spec. it should not be used here.

we need to refactor to have a `size` prop and another prop for style (the description of `type` should be avoided at fiori-fundamentals layer.

File: src/Button/Button.Component.js [hunk 4]
```diff
@@ -200,6 +214,11 @@ content like appearing in a table or list. There may be more than one on the pag
                         Reject
                     </Button>
 
+                    <Button glyph='alert' option='emphasized'
+                        type='medium'>
```
[the remark this answers is not in the document]
Comment (leannerivera) [reply src/Button/Button.Component.js]: Hi Greg, I fixed this. I was just following the patterns shown in the previous buttons. It should be fixed now.
